### PR TITLE
Feature/extract email settings into own menu item

### PR DESCRIPTION
**Pull request by ulferts**
Consists of two parts:
* Extracting the two email related tabs from the general settings admin menu into their own menu structure. One could probably now restructure the individual fieldsets into tabs but I did not go to that length.
* Optimizing the language specific text area fields to only use one ckeditor powered text area instead of having one such field per language which led to a noticeable delay when opening a page having such a field: https://community.openproject.com/projects/openproject/work_packages/32136

### CODE REVIEW

**Comment by openprojectci** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/opf/openproject/blob/3e35ff9583fccc7be3b1f908b34e6163a829bb81/frontend/src/app/ckeditor/ckeditor-augmented-textarea.component.ts#L43">frontend/src/app/ckeditor/ckeditor-augmented-textarea.component.ts#L43</a> - Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: config/routes.rb [hunk 1]
```diff
@@ -384,11 +384,16 @@
     end
   end
 
-  # We should fix this crappy routing (split up and rename controller methods)
-  get '/settings' => 'settings#index'
-  scope 'settings', controller: 'settings' do
-    match 'edit', action: 'edit', via: %i[get post]
```
Comment (HDinger) [inline config/routes.rb:390]: When trying to update the system settings (e.g. by changing the Welcome Block) I get an 404 `No route matches [POST] "/settings/edit"`.

**Review by HDinger — COMMENTED** [review]
Looks good 👍 Please have a look at my comment below.

**Comment by ulferts** [comment]
Blatant omission on my side. I only adapted it on one tab and then simply forgot to adapt the others. Thanks for catching, @HDinger.

**Review by HDinger — APPROVED** [review]

**Comment by HDinger** [comment]
Please have a look at the failing tests. Afterwards good to merge 👍

**Comment by ulferts** [comment]
@HDinger I adapted the wrong path (it had been added to dev in the meantime). Interestingly, the errors did not occur locally and they indicate, that some tests are not properly isolated as some test seems to set the hostname/protocol setting.

**Comment by HDinger** [comment]
@ulferts There is still one test failing.

**Pull request merged**